### ISSUE TITLE
Docker image for lite app

### ISSUE THREAD

**Issue by MichaelWarburton-Flutterint** [HemmeligOrg/lite.hemmelig.app#1]
For self-hosting it would be nice if we could have a docker image for the lite app. Could be added to the Docker Compose file, and the link in the main app point to the self-hosted lite app?

**Comment by bjarneo** [comment]
A docker image could be created. The only thing is that this service is not maintained.

**Comment by MichaelWarburton-Flutterint** [comment]
Should the lite app link in the main self-hosted image just be removed then?

**Comment by bjarneo** [comment]
No, even though it is not maintained anymore does not mean it is not working.
